wall_e/src/resources/cogs/ban.py

```diff
@@ -0,0 +1,343 @@
+from discord.ext import commands
+import discord
+from resources.utilities.embed import embed as em
+from WalleModels.models import BannedUsers, BanRecords
+import datetime
+import pytz
+import re
+from typing import Union
+import logging
+from asgiref.sync import sync_to_async
+logger = logging.getLogger('wall_e')
+
+
+class Ban(commands.Cog):
+
+    def __init__(self, bot, config):
+        self.bot = bot
+        self.config = config
+        self.blacklist = []
+        self.errorColour = 0xA6192E
+
+    @sync_to_async
+    def insert_ban(self, username, user_id):
```

Comment:
all the method that deal with the database interactions need to do under the same `.py` file that contains the modes. similiar to how I did it for the `Reminder` and `CommandStat`